code/game/machinery/doors/door.dm

```diff
@@ -137,64 +147,57 @@
 	if(density)
 		flick("door_deny", src)
 
-
 /obj/structure/machinery/door/attackby(obj/item/I, mob/user)
 	if(!(I.flags_item & NOBLUDGEON))
 		try_to_activate_door(user)
-		return 1
+		return TRUE
 
 /obj/structure/machinery/door/emp_act(severity)
-	if(prob(20/severity) && (istype(src,/obj/structure/machinery/door/airlock) || istype(src,/obj/structure/machinery/door/window)) )
-		open()
+	if(prob(20/severity))
+		if((istype(src, /obj/structure/machinery/door/airlock) || istype(src, /obj/structure/machinery/door/window)))
+			open()
 	if(prob(40/severity))
 		if(secondsElectrified == 0)
 			secondsElectrified = -1
 			spawn(30 SECONDS)
 				secondsElectrified = 0
 	..()
 
-
 /obj/structure/machinery/door/ex_act(severity)
-	if(unacidable) return
+	if(unacidable)
+		return
 
 	if(density)
 		switch(severity)
 			if(0 to EXPLOSION_THRESHOLD_LOW)
 				if(prob(80))
-					var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
-					s.set_up(2, 1, src)
-					s.start()
+					var/datum/effect_system/spark_spread/spark = new /datum/effect_system/spark_spread
+					spark.set_up(2, 1, src)
+					spark.start()
 			if(EXPLOSION_THRESHOLD_LOW to INFINITY)
 				qdel(src)
 	else
 		switch(severity)
 			if(0 to EXPLOSION_THRESHOLD_MEDIUM)
 				if(prob(80))
-					var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
-					s.set_up(2, 1, src)
-					s.start()
+					var/datum/effect_system/spark_spread/spark = new /datum/effect_system/spark_spread
+					spark.set_up(2, 1, src)
+					spark.start()
 			else
 				qdel(src)
 	return
 
-
 /obj/structure/machinery/door/get_explosion_resistance()
 	if(density)
 		if(unacidable)
-			return 1000000
+			return 1000000 //I have no idea why do we use magic number here...
```

Comment:
I am making a big assumption here but I'm guessing this is just here to be a "big number" in order to negate explosions. I reckon it's just been copypasted ever since.
https://github.com/cmss13-devs/cmss13/blob/4ff511287574c8613cb080d5038c8693b3d849b8/code/__DEFINES/misc.dm#L35

Explosion max power define is 5000 and looks like it's used in this calculation.

https://github.com/cmss13-devs/cmss13/blob/4ff511287574c8613cb080d5038c8693b3d849b8/code/datums/autocells/explosion.dm#L168-L172

Reply:
Yup, sounds like something defines would be good at, I will leave a comment for now, maybe someone will take care of this, hopefully.